templates/restic_script_Linux.j2

```diff
@@ -102,6 +102,39 @@ if [[ -z ${CRON+x} ]]; then
 else
   MODE_TAG="--tag cron"
 fi
+
+
+{% if item.enable_logging %}
+if [[ $({% if item.stdin is defined and item.stdin == true %}
+  {{ item.stdin_cmd }} | {{ restic_install_path }}/restic backup \
+    --stdin $MODE_TAG \
+    {{ tags(item.tags) }} \
+    {{ stdin_filename(item.stdin_filename) }} \
+    {% if item.exclude is defined %}{{ exclude(item.exclude) }}{% endif %} \
+    $@
+{% else %}
+  {{ restic_install_path }}/restic backup $BACKUP_SOURCE $MODE_TAG \
+    {{ tags(item.tags) }} \
+    {% if item.exclude is defined %}{{ exclude(item.exclude) }}{% endif %} \
+    $@
+{% endif %}) ]]
+then
+    echo "$(date +%Y-%m-%d) OK" >> {{ restic_log_dir }}/{{ item.name }}-backup-result.log
+else
+    echo "$(date +%Y-%m-%d) ERROR" >> {{ restic_log_dir }}/{{ item.name }}-backup-result.log
+fi
+{#
+  Define stdin forget commands
+#}
+if [[ $({{ restic_install_path }}/restic forget --path {{ path(item) }} {{ retention_pattern(item) }} {% if item.prune is defined and item.prune == true %}--prune{% endif %}) ]]
+then
+    echo "$(date +%Y-%m-%d) OK" >> {{ restic_log_dir }}/{{ item.name }}-forget-result.log
```

Comment:
Would it be possible to make the date format configurable via variable? This role allows for cronjobs to also be set hourly or even less, which makes a fixed Y-m-d timestamt limiting.

Reply:
It sure would be, but I’m afraid it would be quite error prone: should the user enter an invalid value the script could fail. 

I think it would be better to add hour, minute and second (maybe fixed to the UTC time zone to avoid DST issues). If hours/minutes/seconds are not relevant in the particular use case you could just ignore them.

Reply:
I changed the date command to `date -u '+%Y-%m-%d %H:%M:%S'`, which outputs something like `2020-10-29 11:46:56` in the UTC time zone.

Reply:
That works, too. If necessary, we can always introduce confugurability later.